AEPLifecycle/Sources/LifecycleDataStoreCache.swift

```diff
@@ -0,0 +1,83 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+
+/// Lifecycle DataStore Cache layer for persisting the timestamp values required for Lifecycle session computation in XDM,
+/// including close timestamp to be used for app close time approximation, start and pause timestamps.
+class LifecycleDataStoreCache {
+
+    static let LOG_TAG = "LifecycleDataStoreCache"
+    private var closeTimeStampSec = TimeInterval()
+    private var lastClosePersistedTimeStampSec = TimeInterval()
+
+    #if DEBUG
+        var dataStore: NamedCollectionDataStore
+    #else
+        private(set) var dataStore: NamedCollectionDataStore
+    #endif
```

Comment:
Where is this being used? For unit testing we can use the constructor.

Reply:
oh yes, that is a correct. Let me remove this.